src/rest/src/main/java/org/geoserver/rest/RestInterceptor.java

```diff
@@ -23,4 +24,14 @@ public boolean preHandle(
 
         return true;
     }
+
+    @Override
+    public void postHandle(
+            HttpServletRequest request,
+            HttpServletResponse response,
+            Object handler,
+            ModelAndView modelAndView)
+            throws Exception {
+        super.postHandle(request, response, handler, modelAndView);
+    }
```

Comment:
This doesn't seem to be doing anything useful?

Reply:
True, leftover from a failed approach.